test/AArch64/go_dwarf.test

```diff
@@ -0,0 +1,18 @@
+# Check that the llvm-bolt update the dwarf information correctly in case:
+#  - DW_AT_low_pc is nullified in case the DW_AT_ranges tag already exists
+#  - DW_AT_high_pc is in the form of DW_FORM_addr
+
+RUN: llvm-bolt %p/Inputs/gcgort_aarch64 -o %t -update-debug-sections
```

Comment:
Can you please also add checks for the values in the original binary? So that we know they are updated correctly.

Reply:
Sure, done